src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/MakeAnApplicationTypesProvider.java

```diff
@@ -139,6 +143,17 @@ && isAcceleratedDetainedAppeal(asylumCase)) {
                     }
                 }
 
+                if (hasHomeOfficeRole && isAcceleratedDetainedAppeal(asylumCase)
+                        && currentState != PENDING_PAYMENT
```

Comment:
Can we please go through the list of excluded states here?

Reply:
After confirmation with Elli, the "Adjourn" and "Expedite" option will not include these interim states (PENDING_PAYMENT, AWAITING_REASONS_FOR_APPEAL, AWAITING_CLARIFYING_QUESTIONS_ANSWERS, AWAITING_CMA_REQUIREMENTS and REASONS_FOR_APPEAL_SUBMITTED)